docs/DeveloperGuide.md

```diff
@@ -210,14 +245,14 @@ is `/create`, the `WorkoutCommand#execute()` method will call `WorkoutList#creat
 to create a new workout in the application. 
 To view the details of the `WorkoutCommand#execute()`, click [here](https://github.com/AY2122S2-CS2113T-T09-2/tp/blob/master/src/main/java/commands/WorkoutCommand.java). 
 <br><br>
-When all methods except the `listAllWorkout()` method is executed, the appropriate
-`FileManager` and `UI` classes will call the appropriate methods depending on the command action.
-From the previous example, the `/create` workout action will call the 
+When all methods except the `listAllWorkout()` method is executed, the
```

Comment:
Perhaps 'is' can be changed to 'are'? Since we're saying that all other methods are executed.